ape_safe/packages.py

```diff
@@ -0,0 +1,125 @@
+from enum import Enum
+from importlib import resources
+from typing import TYPE_CHECKING, Union
+
+import requests
+from ape.managers.project import ProjectManager
+from packaging.version import Version
+from pydantic import BaseModel
+
+if TYPE_CHECKING:
+    from ape.contracts import ContractContainer, ContractInstance
+
+with resources.as_file(resources.files(__package__).joinpath("manifests")) as manifest_folder:
```

Comment:
It would be better for module loading if you could make some of this lazier in its loading

Reply:
42343fa